### PR TITLE
Feature/4169 task info screen

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
New information screen for the contact diary
Legal texts are placeholders at the moment, they get changed in a separate PR later

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4169

## Screenshots
Simulator Screen Shot - iPhone 12 Pro - 2020-12-10 at 16 46 34
Simulator Screen Shot - iPhone 12 Pro - 2020-12-10 at 16 46 57

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -1050,6 +1050,25 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "Home_Alert_RiskStatusLowered_PrimaryButtonTitle" = "OK";
 
+/* Contact Diary Information */
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1053]: There is a section at the bottom of the File for all Strings related to ContactDiary, would be nice to cluster them :)

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 2]
```diff
@@ -688,6 +688,24 @@ enum AppStrings {
 		static let riskStatusLoweredAlertPrimaryButtonTitle = NSLocalizedString("Home_Alert_RiskStatusLowered_PrimaryButtonTitle", comment: "")
 	}
 
+	enum ContactDiaryInformation {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:691]: Same as with the stringsFile, there is a section for ContactDiary :)

**Review by marcussc — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: I moved them

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift (hunk 2 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:691]: Moved as well

**Review by marcussc — APPROVED** [review]

**Comment by tcfos** [comment]
Check the vertical alignment of the icons and the texts. The "Lock" icon is currrently placed in the middle of the vertical height of the corresponding text block. should move up – the Figma mockup places the icon close to the top of the text block.

**Review by janetback — CHANGES_REQUESTED** [review]
Hi, the architect informed us yesterday that it's 16 days, not 14,
Could you therefore please change sentence in string file to:
Ihre Einträge werden nur auf Ihrem Smartphone gespeichert und nach 16 Tagen automatisch gelöscht. Thanks!

**Review by janetback — APPROVED** [review]
UA approved

**Pull request merged**